### PR TITLE
Convert scripts to the format used by console_scripts

### PR DESCRIPTION
**Pull request by segevfiner**
@kislyuk

Split from #474

### CODE REVIEW

**Comment by segevfiner** [comment]
`ruff` seems to have made a backwards incompatible change that broke your CI.

**Comment by segevfiner** [comment]
@kislyuk Did you look at this? Want me to make a separate PR to fix CI?

File: argcomplete/scripts/activate_global_python_argcomplete.py [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+import argparse
+import os
+import shutil
+import site
+import subprocess
+import sys
+
+import argcomplete
+
+__package__ = "argcomplete"
```
Comment (kislyuk) [inline argcomplete/scripts/activate_global_python_argcomplete.py:20]: Why is this set? Please either document a reason why this needs to be set in a comment, or remove this line.

**Review by kislyuk — CHANGES_REQUESTED** [review]
@segevfiner thanks for your contribution, but can you please change the git history such that the move is explicitly reflected? In other words, `git mv` the files from one place to the other, then add new files in their place with no commonality with the original files.

**Comment by segevfiner** [comment]
> @segevfiner thanks for your contribution, but can you please change the git history such that the move is explicitly reflected? In other words, `git mv` the files from one place to the other, then add new files in their place with no commonality with the original files.
> 
> I can't review this PR as it is because there is no diff reflecting any changes (or lack thereof) made in the move.

Just an FYI: A `git mv` is just a `git rm` + `git add` of the new file. There is no special metadata saved in Git about moves, it's always a heuristic. So you might still need to review the individual commits.

File: argcomplete/scripts/activate_global_python_argcomplete.py (hunk 1 above)
Comment (segevfiner) [reply argcomplete/scripts/activate_global_python_argcomplete.py:20]: Added a comment

**Comment by segevfiner** [comment]
@kislyuk Fixes committed.

**Comment by kislyuk** [comment]
Thanks, merged

**Pull request merged**